charts/lab-partner/templates/ingress.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: lab-partner
+  labels:
+    {{- include "lab-partner.labels" . | nindent 4 }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  rules:
+    {{- range .Values.ingress.hosts }}
```

Comment:
I don't really see a point in specifying multiple hosts here.  we're only ever going to have one host and one path.  could you simplify this?